editor/editor_export.cpp

```diff
@@ -918,14 +930,21 @@ Error EditorExportPlatform::export_project_files(const Ref<EditorExportPreset> &
 				} else {
 					export_plugins.write[i]->_export_file(path, type, features);
 				}
+				err = OK;
```

Comment:
This doesn't seem useful, I don't see any check for `err != OK` in this loop which would not have an assignment to `err` before it.